subtests/docker_cli/run_volumes_selinux/run_volumes_selinux.py

```diff
@@ -149,7 +136,8 @@ def touch_and_check(self, cont, volume, filename, context_pre,
                       get_selinux_context(volume))
         self.logdebug("Touching /tmp/test/%s in container"
                       % filename)
-        os.write(cont.stdin, "touch /tmp/test/%s\necho RET: $?\n" % filename)
+        # Don't just update directory entries with touch, add content too
```

Comment:
Why? Since I don't understand why this is important, maybe the comment should include a reason? Also, the debug log above should change to 'Creating'.